### ISSUE TITLE
aria-label should not be used on <time>

### ISSUE THREAD

**Issue by AlecRust** [basecamp/local_time#122]
As-per [MDN docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label#associated_roles):

> The `aria-label` attribute is NOT supported in code, (..) and `time`
>
> The `aria-label` attribute is intended for interactive elements only. When placed on non-interactive elements, such as those listed above, it may not be read or may confuse your users as a non-interactive element that acts like an interactive one.

This causes some errors with accessibility tools that check `<time>` doesn't have `aria-label`. Looks like this gem adds `aria-label`, but it should probably be something like `data-label` instead.

Looks like this was added [here](https://github.com/basecamp/local_time/commit/78244c263843164b70b746b3029c7ead1e931e83) to improve accessibility, but I don't think it's doing that. `<time>` element is all the description that is needed.

**Comment by josefarias** [comment]
You're right, thanks for calling this out! https://github.com/basecamp/local_time/pull/140
